src/styles/layout.css

```diff
@@ -0,0 +1,14 @@
+.flexbox {
```

Comment:
I think this css is not used. Please check.

Reply:
Yes I have left only the class `.bottom_center`